examples/tfengine/generated/folder_foundation/audit/terraform.tfvars

```diff
@@ -13,5 +13,16 @@
 # limitations under the License.
 
 
-folder         = "folders/12345678"
-auditors_group = "example-auditors@example.com"
+folder            = "folders/12345678"
+auditors_group    = "example-auditors@example.com"
+filter            = "logName:\"logs/cloudaudit.googleapis.com\" OR logName=\"logs/forseti\" OR logName=\"logs/application\""
```

Comment:
Minor nit, I find it hard to parse with all these backslash escapes, would it be possible to replace it with a heredoc string? As in
```
filter = <<-EOF
logName:"logs/cloudaudit.googleapis.com" OR logName="logs/forseti" OR logName="logs/application"
EOF
```

Reply:
I don't think it's possible because they have to land the config that way 'cause it is passed to be passed with the `"` escaped, but I think I maybe lack of context.

Wait to see what @xingao267 says

Reply:
If I'm understanding correctly the escapes are just because of HCL syntax using double quotes for strings. In the API itself I don't think it needs escapes, it just takes the string with the quotes. So I think using EOF heredoc is equivalent to using quotes with escapes.

Reply:
Tbh I'm not sure. The schema says that it should be kept, [look](https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/blob/master/docs/tfengine/schemas/audit.md).

Is this somehow tested by the unit/integration tests? I can save some time if that's true, otherwise I think I'll have to test it manually.

Reply:
I think our docs are subtly wrong here. The escapes are just for Terraform, not for the API. Looking at the sample queries in https://cloud.google.com/logging/docs/view/query-library, it looks like the API expects normal quotes, not escaped quotes, and we're only escaping them since we're wrapping them in `""`. Using EOF/heredocs would make it more obvious what the final filter string is.

Reply:
From what I can tell it's not tested by the integration tests. They don't set up any modules which use additional_filters.

Reply:
I applied the changes, but I'm not sure if we're in the same situation as when this discussion appeared, 'cause now the `additional_filters` is a `list(string)` instead of `string`. Therefor, the `EOF` thing doesn't apply.

Anyway, I did [this commit](https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/pull/948/commits/84097ff3272c67f1ca4f079bb1a20a794191bf0a) with what I think is correct according to what we talked

Let me know if I'm misunderstanding something, so I can revert the commit and change it

Reply:
Nevermind. Test are not passing because of it.

Actually, now that it is a `list(string)`, I think we're good to go as it was previously.

Just confirm and I'll remove that commit. We can discuss it in our meeting

Reply:
Yes, I think with `list(string)` it should be left as it was. In the recipes we need to use double escapes because during the conversion to .tf files strings get escaped, so in .hcl we write `\\\"`so that it becomes `\"` in .tf.

Reply:
Got it.
I'm resolving and reverting the commit